modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -6258,25 +6259,23 @@ public final void setNodeOrientation(NodeOrientation orientation) {
     }
 
     public final NodeOrientation getNodeOrientation() {
-        return nodeOrientation == null ? defaultNodeOrientation : nodeOrientation.get();
+        return nodeOrientation == null ? DEFAULT_NODE_ORIENTATION : nodeOrientation.get();
     }
 
     /**
-     * Property holding NodeOrientation.
-     * <p>
      * Node orientation describes the flow of visual data within a node.
      * In the English speaking world, visual data normally flows from
      * left-to-right. In an Arabic or Hebrew world, visual data flows
-     * from right-to-left.  This is consistent with the reading order
-     * of text in both worlds.  The default value is left-to-right.
-     * </p>
+     * from right-to-left. This is consistent with the reading order
+     * of text in both worlds.
      *
-     * @return NodeOrientation
+     * @defaultValue if the system property {@code javafx.scene.nodeOrientation.RTL} is {@code true},
```

Comment:
Restore the `@return` to avoid a warning.

Reply:
it seems that placing javadoc at the property declaration instead of the method which returns said property eliminates the warning. 
See for example Stage:1329

Reply:
> I suggest to move the doc to the field.

Yeah that's even better.